lib/screens/account_pharse_confirmation/account_pharse_confirmation_screen.dart

```diff
@@ -0,0 +1,118 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:sunshine/core/core.dart';
+import 'package:sunshine/models/models.dart';
+import 'package:sunshine/sunshine.dart';
+import 'account_pharse_confirmation_view_model.dart';
+
+class AccountPharseConfirmationScreen extends StatefulWidget {
+  const AccountPharseConfirmationScreen({
+    Key key,
+    this.accountBackup,
+  }) : super(key: key);
+
+  final AccountBackup accountBackup;
+
+  @override
+  _AccountPharseConfirmationScreenState createState() =>
+      _AccountPharseConfirmationScreenState();
+}
+
+class _AccountPharseConfirmationScreenState
+    extends State<AccountPharseConfirmationScreen> {
+  final _pharseTextController = TextEditingController();
+  String _errorText;
+  @override
+  Widget build(BuildContext context) {
+    return BaseWidget<AccountPharseConfirmationViewModel>(
+      child: AppBar(
+        backgroundColor: Colors.transparent,
+        elevation: 0,
+        centerTitle: false,
+        leading: BackButton(
+          color: Colors.black,
+        ),
+        title: Text(
+          'Account Pharse',
```

Comment:
```suggestion
          'Account Phrase',
```
I'm also inclined to FIND and REPLACE all the variable names like the class name from `Pharse` -> `Phrase` but that's not as important because it's not user facing...